testsuite/systemtest_long/qemu-openasip/README.md

```diff
@@ -0,0 +1,10 @@
+# tcetest_qemu-openasip.py
+
+tcetest_qemu-openasip_integration.py compiles and runs a small program in QEMU
+that uses custom OSAL instructions. You should have built QEMU-OpenASIP beforehand,
+for example with a script from tools/scripts/install_qemu.py
+
+By default the test assumes that QEMU is installed in $HOME/qemu-openasip, but
+a new path can be provided using QEMU_OPENASIP_PATH environment variable
```

Comment:
Perhaps flip the name to OPENASIP_QEMU_PATH since it's OpenASIP's QEMU path, not QEMU's OpenASIP path.